### DISCUSSION TITLE
Accessibility option to provide a familiarity?

### DISCUSSION THREAD

**Discussion by hyiltiz** [helix-editor/helix#1697] [Ideas] [answered]
`oh-my-zsh`, emacs `viper` etc. provide an option to set the how advanced the users' think they are w.r.t. the tool. Something like that might help offer a smoother transition experience for new users. For example, `set accessibility = vim` might make bindings such as `:b` (short for `:buffers`) available which becomes muscle memory to most. I had to dig through the source to find that `SPC b` is the bindings for helix. While helix is very discoverable, not even having to discover might be even better. We can help the transition by showing `SPC b` when user types `:b` as its alias when the accessibility mode is active.

**Comment by LGUG2Z** [comment, marked as answer]
https://github.com/LGUG2Z/helix-vim

I decided to publish my attempt at trying to implement a set of key bindings that allow me to maximize the use of my Vim muscle memory. 🎉
